bin/install.js

```diff
@@ -266,5 +266,5 @@ function* updateDependencies(root, pkgs, propName, saveExact) {
     newDeps[key] = deps[key];
   }
   pkg[propName] = newDeps;
-  yield fs.writeFile(pkgFile, JSON.stringify(pkg, null, 2));
+  yield fs.writeFile(pkgFile, JSON.stringify(pkg, null, 2) + '\n');
```

Comment:
不一定的

Sent from my iPhone

> On Sep 2, 2016, at 10:36 AM, Yiyu He <notifications@github.com> wrote:
> 
> In bin/install.js:
> 
> > @@ -266,5 +266,5 @@ function* updateDependencies(root, pkgs, propName, saveExact) {
> >      newDeps[key] = deps[key];
> >    }
> >    pkg[propName] = newDeps;
> > -  yield fs.writeFile(pkgFile, JSON.stringify(pkg, null, 2));
> > +  yield fs.writeFile(pkgFile, JSON.stringify(pkg, null, 2) + '\n');
> require('os').EOL
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub, or mute the thread.
>